dashboard/app/views/shared/_maybe_set_hoc_secret.html.haml

```diff
@@ -0,0 +1,12 @@
+- if current_user && current_user.admin?
+  - # For admins, set a client-side secret that allows them to access restricted HOC levels.
+  :javascript
+    //<![CDATA[
+    localStorage.setItem('hoc_secret', '#{DCDO.get("hoc_secret", nil)}');
```

Comment:
This would throw an exception for admins who are (1) logged in as an admin and (2) in Safari private browsing mode—but that seems like an unlikely scenario over this weekend.

To test the script in Safari private browsing mode, might need to log in as an admin. If this throws an error, would it potentially cause the rest of the page's top-level scripts not to execute? If not, seems fine. If it would cause trouble, might be worth just throwing this line in a try / catch